clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleServerGroup.groovy

```diff
@@ -110,8 +113,11 @@ class GoogleServerGroup {
     Set<String> getLoadBalancers() {
       Set<String> loadBalancerNames = []
       def asg = GoogleServerGroup.this.asg
-      if (asg && asg.containsKey("loadBalancerNames")) {
-        loadBalancerNames = (Set<String>) asg.loadBalancerNames
+      if (asg && asg.containsKey(REGIONAL_LB_NAMES)) {
+        loadBalancerNames.addAll(asg.get(REGIONAL_LB_NAMES) as Set<String>)
+      }
+      if (asg && asg.containsKey(GLOBAL_LB_NAMES)) {
```

Comment:
This (and the above) can by groovy-fied into `asg?.containsKey(GLOBAL_LB_NAMES)`